### ISSUE TITLE
ValueError: Config parameter "html" does not exist.

### ISSUE THREAD

**Issue by ibuda** [pandas-profiling/pandas-profiling#375]
First of all, I want to thank you for a great product that you've built.
I came across it early last year, and it worked like a charm. 
Yesterday, I tried it again and when I run this generic line:
```
profile = ProfileReport(data, title='Pandas Profiling Report', html={'style':{'full_width':True}})
```

 I got the following error:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-27-e5a2e28a7045> in <module>
----> 1 profile = ProfileReport(data, title='Pandas Profiling Report', html={'style':{'full_width':True}})

~/anaconda3/lib/python3.7/site-packages/pandas_profiling/__init__.py in __init__(self, df, **kwargs)
     33 
     34     def __init__(self, df, **kwargs):
---> 35         config.set_kwargs(kwargs)
     36 
     37         # Treat index as any other column

~/anaconda3/lib/python3.7/site-packages/pandas_profiling/config.py in set_kwargs(self, kwargs)
     49 
     50         """
---> 51         self._set_kwargs(self.config, kwargs)
     52 
     53     def __getitem__(self, item):

~/anaconda3/lib/python3.7/site-packages/pandas_profiling/config.py in _set_kwargs(self, reference, values)
     39                     reference[key].set(value)
     40             else:
---> 41                 raise ValueError('Config parameter "{}" does not exist.'.format(key))
     42 
     43     def set_kwargs(self, kwargs) -> None:

ValueError: Config parameter "html" does not exist.
```

**To Reproduce** One can reproduce this error by running the following code:
```
from sklearn.datasets import load_digits
digits = load_digits()
X = pd.DataFrame(digits.data)
profile = ProfileReport(X, title='Pandas Profiling Report', html={'style':{'full_width':True}})
```

**Version information:**
python: 3.7.3
pandas_profiling: 2.4.0

All other required modules, I made sure and `pip install -r`-ed the requirements.txt.

**Comment by ibuda** [comment]
Ok, I have managed to resolve the issue and this is what I did.

1. I re-installed the package using `pip install pandas-profiling[notebook,html] (the previous installation was with conda) and the error disappeared. However, another issue appeared once I called profile in jupyter lab:
![Screenshot from 2020-02-11 11-39-21](https://user-images.githubusercontent.com/42291858/74225331-2cab9f00-4cc3-11ea-9605-77f417cc5c65.png)
2. Lurking through the issues list, I found this solution to the issue from 1., it works like a charm:
`profile.to_notebook_iframe()`

Closing the issue.
